### DISCUSSION TITLE
cats dependency

### DISCUSSION THREAD

**Discussion by tnielens** [pityka/saddle#364] [Q&A] [answered]
I understood that this fork rebased saddle-core on `cats.kernel.Order` instead of Ordering for the std lib. 
Are there other uses of cats aside from Order in saddle? Or plans to extend the library and implement other cats typeclasses, Functor, Monoid, etc?

**Comment by pityka** [comment, marked as answer]
The rational is that saddle needs an ordering typeclass which is specialized for the primitives. There is no other use of cats, neither I plan to add support. I don't oppose to add cats type classes in a separate (saddle-cats) module. 
I picked cats kernel for the ordering typeclass for stability and widespread use in the ecosystem.

(https://github.com/scala/scala/blob/2.13.x/src/library/scala/math/Ordering.scala#L73)
(https://github.com/typelevel/cats/blob/main/kernel/src/main/scala/cats/kernel/Order.scala#L22)
